easytensor/easytensor.cabal

```diff
@@ -38,6 +38,8 @@ library
         Numeric.DataFrame.ST
         Numeric.Matrix
         Numeric.Matrix.Class
+        Numeric.Matrix.Mat44d
```

Comment:
In general, orphan instances are a very bad thing, so I try to hide them in `other-modules` if possible.
I suggest to do the same as in quaternion case: import modules with orphan instances in `Numeric.Matrix` and remove them from `exposed-modules`. I am not 100% sure about `Numeric.Matrix.Class` in this regard: it compares to `Numeric.Quaternion.Class`, but has more stuff inside; maybe some helper functions should still be exposed there and not re-exported in `Numeric.Matrix`?..